### ISSUE TITLE
Update `@yamada-ui/resizable` accessibility

### ISSUE THREAD

**Issue by nakayan5** [yamada-ui/yamada-ui#1633]
### Description

his issue aims to address accessibility deficiencies in our resizable component as per W3C guidelines.
The component currently does not meet the accessibility standards outlined in the W3C documentation.

The goal is to ensure the component is accessible to all users, including those using assistive technologies.
Ensure that all unchecked items in this checklist should be addressed.

Keyboard Interaction
- [ ] ~~Enter: If the primary pane is not collapsed, collapses the pane. If the pane is collapsed, restores the splitter to its previous position.~~

WAI-ARIA Roles, States, and Properties
- [ ] If the primary pane has a visible label, it is referenced by [aria-labelledby](https://w3c.github.io/aria/#aria-labelledby) on the separator element. Otherwise, the separator element has a label provided by [aria-label](https://w3c.github.io/aria/#aria-label).

###  Problem Statement/Justification
none

###  Proposed Solution or API
No response

###  Alternatives
none 

### Are you willing to participate in implementing this feature and create a pull request include the implementation?
None

###  Additional Information

related https://github.com/yamada-ui/yamada-ui/issues/1514

**Comment by cidkumagai** [comment]
I would like to challenge this issue.
